tests/test_georaster.py

```diff
@@ -504,3 +504,26 @@ def test_value_at_coords(self):
         assert img.value_at_coords(x, y) == \
             img.value_at_coords(lon, lat, latlon=True) == \
             img.data[0, -1, -2]
+
+    def test_from_array(self):
+
+        # Test that from_array works if nothing is changed
+        # -> most tests already performed in test_copy, no need for more
+        img = gr.Raster(datasets.get_path('landsat_B4'))
+        out_img = gr.Raster.from_array(img.data, img.transform, img.crs, nodata=img.nodata)
```

Comment:
If I understand `Raster.from_array()` correctly, `data` is not copied, so the below assertion will always be true, because it is the same object. I might be wrong though...

Reply:
What do you mean? img and out_img are pointing to different objects. For example, if I do:
```
out_img.data += 5
np.array_equal(img.data, out_img.data)
```
returns False.
Or did I misunderstand your statement?

Reply:
Oh, it must be copying implicitly somewhere, which I think is the preferred way.
Maybe here?:

https://github.com/GlacioHack/GeoUtils/blob/f1641a8182713389aaec8e1c2fdd87ad4c47bccb/geoutils/georaster.py#L265

Then everything is fine!